### DISCUSSION TITLE
kindling probe init err: bpf_load_program() err=13

### DISCUSSION THREAD

**Discussion by Dingzhen1233** [harmonycloud/kindling#355] [agent probe] [answered]
1、按照FAQ中的内容重新打了kindling-agent的镜像
2、部署之后pod持续报错重启：

R3 type=map_value expected=fp
kindling probe init err: bpf_load_program() err=13 event=finish_task_switch message=0: (bf) r8 = r1
1: (b7) r1 = 0
2: (63) *(u32 *)(r10 -248) = r1
3: (bf) r2 = r10
4: (07) r2 += -248
5: (18) r1 = 0xffffa0a699967400
7: (85) call bpf_map_lookup_elem#1
8: (bf) r9 = r0
9: (15) if r9 2022/11/16 12:03:31 Failed to run application: failed to start application: fail to init probe

**Comment by Dingzhen1233** [comment, marked as answer]
转到issue了，不在此讨论问题了

**Comment by jundizhou** [comment]
请提供下具体的操作系统和内核版本

**Reply by Dingzhen1233** [reply]
Ubuntu 18.04.2 LTS

Linux dggphicprd30663 4.15.0-45-generic #48-Ubuntu SMP Tue Jan 29 16:28:13 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

**Reply by jundizhou** [reply]
你可以先注释 agent-libs中的driver/bpf/probe.c的第284-360行，已知在内核版本小于4.17并且大于3.10的版本中存在该问题

**Reply by Dingzhen1233** [reply]
多谢多谢，可以正常拉起pod了
